rpc/client/rpc_test.go

```diff
@@ -252,7 +252,8 @@ func TestAppCalls(t *testing.T) {
 func TestBroadcastTxSync(t *testing.T) {
 	require := require.New(t)
 
-	mempool := node.MempoolReactor().Mempool
+	// TODO (melekes): use mempool which is set on RPC rather than getting it from node
```

Comment:
Is there an issue to track this?

Reply:
https://github.com/tendermint/tendermint/issues/3433